hoa2dot/core.py

```diff
@@ -262,6 +265,10 @@ def __eq__(self, other):
         """Check equality between two AndLabelExpressions."""
         return isinstance(other, AndLabelExpression) and self.subexpressions == other.subexpressions
 
+    def __hash__(self):
+        """Compute the hash of a AndLabelExpression."""
```

Comment:
shouldn't the docstring be `an AndLabelExpression.` instead of  `a AndLabelExpression.`??